common/app/views/commercial/cards/itemCard.scala.html

```diff
@@ -16,7 +16,7 @@
 <div class="@CssClassBuilder.linkFromStandardCard(item, optAdvertClassNames, optClassNames, useCardBranding)">
     <a href="@item.targetUrl"
        data-link-name="@omnitureId"
-       class="advert">
+       class="advert advert--card-branding">
```

Comment:
Perhaps `advert--branded` would be better here. Would also solve @kelvin-chappell 's comments with `isCardBranding -> isBranded`